rootppl/models/phylogenetics/clads2/junk/attempt-clads2.cu

```diff
@@ -0,0 +1,339 @@
+/*
+ * File clads2.cu defines the ClaDS2 model
```

Comment:
There are several files under a folder `junk`. Should they really be committed? If so, what is the purpose of the `junk` folder?

Reply:
I briefly touched upon this in my PR description. This is one of the reasons why my initial idea was to discuss with Joey, before making the final PR. In a nutshell this is either older reference code, that I felt uneasy to delete, or some promising attempts, that have not been fully tested. They should go in a different branch, I suppose.

Reply:
Comments on how to handle it, appreciated.

Reply:
Many of these files are my old files. There are only two of these I think should stay:

1. `crbd/junk/condbd.cu`
2. `crbd/junk/stack.cuh`

These could turn out to be useful for reference when Daniel want to generate RootPPL code. However, they should eventually be removed as well. A directory for them, for now could be `crbd/generic_stack/`? Feel free to remove the rest of my old phylogenetics reference files that are no longer used. 

I do not know what to do with your files in the `junk` directories.

Reply:
We refactored those.

Reply:
We need to rename the folder and add a good description of what these files are. I do not like to add folders with 'junk' in the repo.